.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: ci-tests
```

Comment:
Perhaps rename this to "ci-test" for simplicity and unification with "lint"/"ci-lint"?